cipher/base64/base64.go

```diff
@@ -0,0 +1,80 @@
+// Package base64
+// base64.go
+// description: The base64 encodage algorithm as defined in the RFC4648 standard.
+// author: [Paul Leydier] (https://github.com/paul-leydier)
+// ref: https://datatracker.ietf.org/doc/html/rfc4648#section-4
+// ref: https://en.wikipedia.org/wiki/Base64
+// see base64_test.go
+package base64
+
+import (
+	"strings" // Used for efficient string builder (more efficient than simply appending strings)
+)
+
+const Alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/"
+
+// Encode encodes the received input bytes slice into a base64 string.
+// The implementation follows the RFC4648 standard, which is documented
+// at https://datatracker.ietf.org/doc/html/rfc4648#section-4
+func Encode(input []byte) (encoded string) {
```

Comment:
```suggestion
func Encode(input []byte) string {
```

Try to avoid the predefined returns.

If you still want to use it then return the variable explicitly like so

```Go
func Something() (val string) {
...
    return val
}
```

Reply:
resolved in https://github.com/TheAlgorithms/Go/pull/437/commits/3ff3e7887f47a7f2e8793bde8d3df9550479dd7d